allauth/socialaccount/providers/oauth2/urls.py

```diff
@@ -1,11 +1,21 @@
 from django.conf.urls import patterns, url, include
 
+from allauth.account import app_settings
+from allauth.socialaccount.providers.oauth2.views import proxy_login_callback
 
 def default_urlpatterns(provider):
     urlpatterns = patterns(provider.package + '.views',
                            url('^login/$', 'oauth2_login',
                                name=provider.id + "_login"),
                            url('^login/callback/$', 'oauth2_callback',
-                               name=provider.id + "_callback"))
+                               name=provider.id + '_callback'))
+
+    if app_settings.LOGIN_PROXY_REDIRECT_WHITELIST:
+        urlpatterns += patterns('',
```

Comment:
This is consistent with both the current package (see call to patterns() in line 7 above) and how we use patterns() in our own backends. Seems better to use this formatting.